### PR TITLE
all: add go:build lines

### PR DESCRIPTION
**Pull request by fjl**
`go1.17 fmt ./...`

### CODE REVIEW

File: accounts/keystore/watch.go [hunk 1]
```diff
@@ -14,6 +14,7 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
 
+//go:build (darwin && !ios && cgo) || freebsd || (linux && !arm64) || netbsd || solaris
```
Comment (karalabe) [inline accounts/keystore/watch.go:17]: wow, paranthesis, amazeballs

**Review by karalabe — APPROVED** [review]
Assume Go's logic is good enough for me not to check it :P

**Comment by Miguelmavs** [comment]
fjl:all-go1.17-buildtag

File: accounts/keystore/watch.go (hunk 1 above)
Comment (Miguelmavs) [reply accounts/keystore/watch.go:17]: fjl:all-go1.17-buildtag

**Review by Miguelmavs — COMMENTED** [review]
fjl:all-go1.17-buildtag

**Comment by Miguelmavs** [comment]
fjl:all-go1.17-buildtag

**Pull request merged**